File: 2010-check-if-word-equals-summation-of-two-words/2010-check-if-word-equals-summation-of-two-words.cpp
class Solution {
private:
    string change(string str) {
        string ch = "";
        for (int i = 0; i < str.length(); ++i) {
            int curr = str[i] - 'a';
            ch += (curr + '0');
        }
        return ch;
    }
public:
    bool isSumEqual(string firstWord, string secondWord, string targetWord) {
        string f, s, t;
        f = change(firstWord);
        s = change(secondWord);
        t = change(targetWord);
        return stoi(f) + stoi(s) == stoi(t);
    }
};

File: 2010-check-if-word-equals-summation-of-two-words/2010-check-if-word-equals-summation-of-two-words.go
func change(str string) string {
    ch := ""
	for _, char := range str {
		curr := int(char - 'a')
		ch += strconv.Itoa(curr)
	}
	return ch
}
func isSumEqual(firstWord string, secondWord string, targetWord string) bool { 
    f := change(firstWord)
    s := change(secondWord)
    t := change(targetWord)

    fInt, _ := strconv.Atoi(f)
	sInt, _ := strconv.Atoi(s)
	tInt, _ := strconv.Atoi(t)
    return fInt + sInt == tInt
}